src/content/structured/components/section-container/guidance.mdx

```diff
@@ -33,9 +33,9 @@ import image5 from "./images/Fig-5-section-containers-align-full-width.png";
 
 ## Introduction
 
-When creating a page, use section containers to group content and align the margins with each other as well as other ICDS components.
+When creating a page, use section containers to group content and align the margins with each other as well as other components.
 
-Section containers are used in ICDS components to control alignment of their content. The following components use section containers with the specified alignments.
+Section containers are used in other components to control alignment of their content. The following components use section containers with the specified alignments.
```

Comment:
Either keep ICDS here or if removing it, don't add 'other' so that it reads "Section containers are used in components to control...".